### ISSUE TITLE
CDN, fallback to local.

### ISSUE THREAD

**Issue by tcastelli** [MiroHibler/meteor-preloader#27]
Hello again!  Due to certain circumstances I've seen how the CDN servers were out, and all my routes depending on that were not loading correctly. Would it be possible to add a fallback place to load them from public folder in that case? (or if anyone has an idea of how to get this working with current functions i'm listening :)

**Comment by MiroHibler** [comment]
Sounds like a good idea. What should be the trigger to switch to public dir? A timeout?

**Comment by tcastelli** [comment]
Yes, i think the current timeout system for sync files would work perfectly to trigger the fallback location (well, maybe the timeout could be set for each file, not sure how much work would that involve).

Update: Well, also getting an error from xmlhttprequest would trigger the event
